pkg/engine/engine.go

```diff
@@ -797,7 +797,6 @@ func (e *Engine) scannerWorker(ctx context.Context) {
 					wgDoneFn: wgDetect.Done,
 				}
 			}
-			continue
```

Comment:
Just double checking but this `continue` looks like a NOP right? @rgmz

Reply:
Correct. It's at the very end of the `for` block and thus has no actual effect.